Library/Homebrew/shims/mac/super/curl-config

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -p
+
+# HOMEBREW_LIBRARY is set by bin/brew
+# HOMEBREW_CURL_CONFIG is set by brew.sh
+# shellcheck disable=SC2154
+source "${HOMEBREW_LIBRARY}/Homebrew/shims/utils.sh"
+
+# Check if HOMEBREW_CURL_CONFIG is set, fallback to system curl-config if not.
+curl_config_exec="${HOMEBREW_CURL_CONFIG:-/usr/bin/curl-config}"
```

Comment:
I don't think we need to introduce `HOMEBREW_CURL_CONFIG`.

```suggestion
```

if you move the `try_exec_non_system` high enough you can hardcode `/usr/bin/curl-config` (or `/usr/bin/${SHIM_FILE}`) elsewhere.

Reply:
> I don't think we need to introduce `HOMEBREW_CURL_CONFIG`.

Agreed.